Include/httpClient/config.h

```diff
@@ -35,6 +35,15 @@
         #define HC_PLATFORM HC_PLATFORM_GDK
     #elif !defined(WINAPI_FAMILY) || (WINAPI_FAMILY == WINAPI_FAMILY_DESKTOP_APP)
         #define HC_PLATFORM HC_PLATFORM_WIN32
+        #include <windef.h>
```

Comment:
isn't this the XAL win32 section?  seems odd to need to put it here

Reply:
moved to correct block